.github/workflows/tensorflow.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         python-version: [3.9.7]
         os: [ubuntu-latest]
-        tf-max: ["2.9", "2.10"]
+        tensorflow-version: ["~=2.8.0", "~=2.9.0", "~=2.10.0"]
```

Comment:
Can this handle minor version releases?

Reply:
`~=2.10.0` is equivalent to saying `>= 2.10.0, == 2.10.*`, which means if a new version of 2.10 is released (e.g. 2.10.1) , then that will be installed if available. In this case I suppose we could use something like `== 2.10.*` instead and will achieve the same result, since we don't really need to specify the minimum minor version here, unless there are some transient dependencies that would mean the latest version doesn't get installed. 

I'm not sure if that's what you meant by 'handle minor version releases'.